site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -4,44 +4,300 @@ title: Upgrade FusionAuth
+
+Q: What is Maintenance Mode? +
+A: Maintenance Mode is a special admin interface for FusionAuth installations in development environments. Maintenance Mode helps you to interactively upgrade FusionAuth or set up database connections. You can switch to Maintenance Mode by switching to the `development` runtime mode and setting Silent Mode to false. Note that Maintenance Mode should never be used in production.
+
+Q: What is Silent Mode? +
+A: link:/docs/v1/tech/guides/silent-mode[Silent Mode] is a feature of FusionAuth that automatically applies database migrations. This is useful for automated deployments. Silent Mode can be enabled in the `development` runtime mode and as of version `1.19.0`, in the `production` runtime mode too. Silent Mode will attempt to perform database migrations on startup. If the database migrations fail, FusionAuth will display an error message on the login page. You should always test your database migrations in a non-production or staging environment before deploying to production.
+
+Q: How many versions can I skip? +
+A: We recommended migrating from one minor version to the next. 
+
+Q: How often should I upgrade? +
+A: You can upgrade whenever you want to. It is your dependency and we won't force you. We do recommend staying relatively close to the latest release. In addition, we don't typically backport bug fixes. If you run into an issue with a bug, you'll need to upgrade to get the fix.
+
+Q: How can I find out about new releases? +
+A: We publish release notes for every release. You can find them here: link:/docs/v1/tech/release-notes[Release Notes]. 
+
+
+== Detailed Upgrade Instructions
+
+This section will guide you with detailed technical instructions on how to upgrade FusionAuth nodes on different platforms.
+
+
+=== Docker
+
+include::docs/v1/tech/shared/_upgrade-using-docker.adoc[]
+
+=== Homebrew
+
+include::docs/v1/tech/shared/_upgrade-using-brew.adoc[]
+
+[NOTE]
+====
+If you want to use manual mode, download the latest version of FusionAuth and follow the instructions below. See https://fusionauth.io/download[Downloads].
```

Comment:
what is manual mode? did you mean maintenance mode?

Reply:
removed the note